engine/src/main/java/org/terasology/config/BindsConfig.java

```diff
@@ -31,23 +30,27 @@
 import com.google.gson.JsonSerializer;
 import org.terasology.engine.SimpleUri;
 import org.terasology.input.Input;
+import org.terasology.input.RegisterBindButton;
 import org.terasology.naming.Name;
 
 import java.lang.reflect.Type;
 import java.util.Arrays;
 import java.util.Collection;
-import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+import static java.util.stream.Collectors.toList;
+
 /**
- * User binds configuration. This holds the key/mouse binding for Button Binds.
+ * The binds configuration holds the mapping from binding uris to Inputs.
+ * The {@link SimpleUri} for a binding contains the module from the binding and the id from the binding annotation, 
```

Comment:
"contains the module from the binding": how can a module be _from_ a binding? Can this be clarified?

You also mention "the id from the binding annotation", I feel this and the previous line could use a little bit of context and further explanation.